python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -48,7 +47,7 @@
 
 
 @experimental
-class AutomationCondition(ABC, DagsterModel):
+class AutomationCondition(ABC):
```

Comment:
unclear if this is the right place to mark `@experiemental` , since that will just emit warnings when any subclass is instantiated but its what caused [1]

Reply:
I think we might want to instead just mark experimental all the static methods on `AutomationCondition` if thats the target public API

Reply:
Yeah that's a good point -- the subclasses are not exported from the top level, so there's no need to explicitly mark them as experimental.